recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.6.26" %}
+{% set version = "1.6.28" %}
 
 package:
   name: libpng
   version: {{ version }}
 
 source:
   fn: libpng-{{ version }}.tar.gz
-  url: ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-{{ version }}.tar.gz
-  sha256: 81bfc8f16ed125622c0c5ad44eeffda19e7a7c7e32f47e43c8932bf32deae7cc
+  url: http://download.sourceforge.net/libpng/libpng-{{ version }}.tar.gz
+  md5: 897ccec1ebfb0922e83c2bfaa1be8748
```

Comment:
I understand that you get this from the original source but `sha256` is safer. I usually download, check the `md5`, and compute the `sha256`.

There is no need for you to do this as I understand it adds some extra tedious steps. Just letting you know how and why the `sha256` was added in the first place.